### PR TITLE
small updates to docker composes and tasks

### PR DESCRIPTION
**Pull request by winprn**

### CODE REVIEW

File: ansible/roles/common/tasks/main.yaml [hunk 1]
```diff
@@ -24,3 +24,20 @@
...
+  shell: "cat /opt/topology/infra/.private_key_seed | tr -d '\n'"
+  register: private_key_seed
+
+- name: Create .env file with environment variables
+  copy:
+    dest: /opt/topology/infra/.env
+    content: |
+      ADDRESSES=/ip4/0.0.0.0/tcp/50000/ws,/ip4/0.0.0.0/tcp/50001
+      BOOTSTRAP=true
+      BOOTSTRAP_PEERS=/dns4/relay.droak.sh/tcp/443/wss/p2p/Qma3GsJmB47xYuyahPZPSadh1avvxfyYQwk8R3UnFrQ6aP
```
Comment (d-roak) [inline ansible/roles/common/tasks/main.yaml:42]: remove this

File: ansible/roles/common/tasks/main.yaml [hunk 2]
```diff
@@ -24,3 +24,20 @@
...
+
+- name: Read the private key seed content directly
+  shell: "cat /opt/topology/infra/.private_key_seed | tr -d '\n'"
+  register: private_key_seed
+
+- name: Create .env file with environment variables
+  copy:
+    dest: /opt/topology/infra/.env
+    content: |
+      ADDRESSES=/ip4/0.0.0.0/tcp/50000/ws,/ip4/0.0.0.0/tcp/50001
```
Comment (d-roak) [inline ansible/roles/common/tasks/main.yaml:40]: remove this

**Review by d-roak — CHANGES_REQUESTED** [review]

File: ansible/roles/common/tasks/main.yaml (hunk 2 above)
Comment (d-roak) [reply ansible/roles/common/tasks/main.yaml:40]: don't rm this

File: ansible/roles/common/tasks/main.yaml [hunk 3]
```diff
@@ -24,3 +24,20 @@
...
+      BOOTSTRAP=true
+      BOOTSTRAP_PEERS=/dns4/relay.droak.sh/tcp/443/wss/p2p/Qma3GsJmB47xYuyahPZPSadh1avvxfyYQwk8R3UnFrQ6aP
+      PRIVATE_KEY_SEED={{ private_key_seed.stdout }}
```
[the remark this answers is not in the document]
Comment (winprn) [reply ansible/roles/common/tasks/main.yaml]: do you want a screenshot? I don't really understand the question

**Review by d-roak — APPROVED** [review]
lgtm

**Pull request merged**